pysat/tests/test_instrument.py

```diff
@@ -1057,8 +1057,7 @@ def test_custom_instrument_load_3(self):
         del test.load
 
         with pytest.raises(AttributeError):
-            pysat.Instrument(inst_module=test, tag='',
-                             clean_level='clean')
+            pysat.Instrument(inst_module=test, tag='', clean_level='clean')
```

Comment:
Yeah, this whole module needs to be cleaned up.  Things like this are present throughout.  I made this change without thinking and decided to commit it so that I wouldn't have to redo it again in the future.